spec/features/refinery/blog/admin/posts_spec.rb

```diff
@@ -338,7 +337,7 @@ module Admin
 
                 click_link "Remove this translation"
 
-                expect(page).not_to have_content(blog_post.title)
+                expect(page).not_to have_content('RU')
```

Comment:
how come this changed away from the post title?

Reply:
Not sure how to test the i18n feature on Refinery (ie, I start `rails s` at `./spec/dummy` and can't find how to enable i18n), but the spec errors seemed to imply that the index shows something like `EN_POST_TITLE EN RU` for each post, with `EN` & `RU` being links to edit that locale's version of the post.

As this test is removing the RU translation, it won't show `RU` after executing the action - but the post still exists, so that's why you still can find the `blog_post.title` there (you get `EN_POST_TITLE EN`).

Reply:
Not really sure how did the previous version work

Reply:
gotcha - that does seem odd that it ever worked 👍